src/utils/diff.js

```diff
@@ -17,12 +33,8 @@ export default (diffProperties, objectOne, objectTwo) => {
       }
 
       return {
-        prev: `${title(prop)}: ${
-          typeof prev === 'string' ? prev.replace(NEW_LINES_REGEX, '') : prev
-        }`,
-        next: `${title(prop)}: ${
-          typeof next === 'string' ? next.replace(NEW_LINES_REGEX, '') : next
-        }`,
+        prev: `${title(prop)}: ${formatValue(prev)}`,
+        next: `${title(prop)}: ${formatValue(next)}`,
```

Comment:
If possible, can we avoid title casing propreties within the release data? It's jarring to have them differ between the full release data view and the diff (and the users who edit the blobs are accustomed to the specific property names).

Reply:
Agreed. We should probably remove it alltogether even from the rules view.